test/go-tests/uniform_test.go

```diff
@@ -0,0 +1,50 @@
+package go_tests
```

Comment:
can we fire a `DELETE` request of the already registered integration at the end of the test as well?

Reply:
sure, I'll add that